static/js/click_handlers.js

```diff
@@ -625,7 +633,7 @@ exports.initialize = function () {
             e.stopPropagation();
             // Cancel editing description if Escape key is pressed.
             if (e.which === 27) {
-                $("[data-finish-editing='.stream-description-editable']").hide();
```

Comment:
You should first make this function generic, remove stream specific code and use `e` to get the class/element name. Make this commit on top of all change.

Reply:
I never thought of it can be used like that, will change it accordingly :+1: